blagues.json

```diff
@@ -2537,5 +2537,107 @@
         "type": "global",
         "joke": "Comment est mort capitaine Crochet ? ",
         "answer": "En se grattant les couilles ! "
-    }
+    },
+    {
+        "id": 463,
+        "type": "global",
+        "joke": "Dark Vador va à la boulangerie",
+        "answer": "-	3 pains et 2 tarte tatin, comme d’habitude ? <br> -	Bonne mémoire <br> -	Facile m’sieur Vador : pain pain pain tarte tatin tarte tatin
```

Comment:
There is some problems with this joke.
- Html is not supported by the api.
- Jokes with multiple lines are not allowed.
- The length is too big